languages/ruby/dyp/lexer_ruby.mll

```diff
@@ -884,7 +884,7 @@ and interp_code start cont state = parse
                incr level;
                tok
            | T_RBRACE _ as tok ->
-               if !level == 0 then begin
```

Comment:
It should be `=` which gets optimized as `==` by the compiler. But:
* it works and I'm not here to rewrite all the bad code
* we can't replace it by `=` because `=` is also forbidden, so we'd have `=*=` or `=|=` which is worse than `phys_equal`.